src/docs/development/frameworks-web/react.mdx

```diff
@@ -43,19 +43,19 @@ Before taking the following steps, you'll want to make sure that you have an exi
 
 ### Material UI
 
-React projects should make use of the [Material-UI](https://mui.com/) component library. Our Brightlayer UI themes are designed to work with these components. You'll also want to install the icons.
+React projects should make use of the [MUI](https://mui.com/) component library. Our Brightlayer UI themes are designed to work with these components. You'll also want to install the icons.
 
 ```sh
-npm install --save @material-ui/core @material-ui/icons
+npm install --save @mui/material @mui/icons-material
```

Comment:
We should include all of the dependencies here required for MUI to work (emotion):
![image](https://user-images.githubusercontent.com/29152776/161317474-e6fce35c-551f-44ef-ad61-be500a11c6c5.png)